### PR TITLE
More understandable description in multisite

### PR DESCRIPTION
**Pull request by luciano-croce**
Currently the labels is:

- Editor
- Default editor for all users

I propose to change the labels to:

- Default editor for current user
- Default editor for all users

This description is much more understandable, IMHO, in multisite envoronment... for example, but also in general, I think.

### CODE REVIEW

**Comment by azaozz** [comment]
Not sure about the string changes. Need to get some more eyes on them.

**Comment by azaozz** [comment]
Thinking we should merge this PR with https://github.com/WordPress/classic-editor/pull/19, then get some more eyes on the string changes. Several strings have changed since these PRs and the tweaks don't apply any more.

I personally like the switch from `'Block Editor'` to `'Edit (Block Editor)'` for the "action links" in the posts list table, and perhaps from `'Editor'` to `'Default Editor'` for the user profile screen.

**Comment by luciano-croce** [comment]
@azaozz I totally agree with that! 👍

- Action Links -> Edit (Block Editor) - Edit (Classic Editor) is more is more comprehensible!

- Also -> "Default Editor" instead "Editor" for the user profile screen is more reliable and understandable in standalone installations.

- In Network Multisite Environment "Default Editor for current user" e/o "Default Editor for current user" for the user profile screen IMHO is the best thing and better understandable for Super Admins and Users.

**Comment by luciano-croce** [comment]
The best IMHO

new-classic-editor-action-links

**Comment by luciano-croce** [comment]
Also consider to add the possibility in Classic Editor to replace Gutenberg (screenshot 1) with Block Editor (screenshot 2) in this screen.

Screenshot 1

wrong-gutenberg-action-links

Screenshot 2

new-replaced-classic-editor-action-links

**Comment by luciano-croce** [comment]
Finally I think appropriate that the Classic Editor restore and enhanced this screen:

Add New (Block Edit)
Add New (Classic Edit)

classic-editor-restored-admin-menu-descriptions

Or alternatively something similar: (for best insert in admin menu without reduce characters horizontal spacing)

Example 1

Add New (Blocks)
Add New (Classic) 

classic-editor-restored-admin-menu-alternative-descriptions

Example 2

Add New (Block)
Add New (Classic)

classic-editor-restored-admin-menu-alternative-descriptions-2

File: classic-editor.php [hunk 1]
```diff
@@ -326,7 +326,7 @@ public static function user_settings() {
 		?>
 		<table class="form-table">
 			<tr class="classic-editor-user-options">
-				<th scope="row"><?php _e( 'Editor', 'classic-editor' ); ?></th>
+				<th scope="row"><?php _e( 'Default editor for current user', 'classic-editor' ); ?></th>
```
Comment (azaozz) [inline classic-editor.php]: Don't think we need "for current user" here, it is implied as this is in the user's profile.

I'm 50/50 on whether "Default editor" is better than just "Editor".

Comment (luciano-croce) [reply classic-editor.php]: @azaozz -> Default Editor (also for me) is more explicative (understandable) of the action that only Editor. ;)

**Pull request merged**